ping-payload-extensions/extensions.md

```diff
@@ -0,0 +1,12 @@
+# This document lists defined extensions
+Some extensions are network specific some are generic
+
+This is a list and short description of all the extensions
+
+
+| Type number  |  Name | Supported sub-networks  | Short Description  | Is this call Required to Implement  |
+|---|---|---|---|---|
+| [0](extensions/type-0.md)  |  Capabilities | All  | Provides a list of enabled types  |  Yes  |
+| [1](extensions/type-1.md)  |  Basic Radius Payload | State, Beacon, Canonical Transaction Index, Transaction Gossip  | Provides the nodes Radius  |  Yes  |
```

Comment:
```suggestion
| [1](extensions/type-1.md)  |  Basic Radius Payload | State, Beacon | Provides the nodes Radius  |  Yes  |
```

Since these other networks don't exist yet, I don't think we should reference them.